apps/server/src/modules/user-login-migration/service/school-migration.service.ts

```diff
@@ -78,6 +78,21 @@ export class SchoolMigrationService {
 		await this.userService.saveAll(notMigratedUsers.data);
 	}
 
+	async restartMigration(schoolId: string): Promise<void> {
+		const school: SchoolDO = await this.schoolService.getSchoolById(schoolId);
+		const migratedUsers: Page<UserDO> = await this.userService.findUsers({
+			schoolId,
+			outdatedSinceEquals: school.oauthMigrationFinished,
+		});
+
+		migratedUsers.data.forEach((user: UserDO) => {
+			user.outdatedSince = undefined;
+		});
+		await this.userService.saveAll(migratedUsers.data);
+
+		school.oauthMigrationMandatory = undefined;
+	}
```

Comment:
That's very, very slow, isn't it?
cann you please log executation time

```suggestion
		console.time("restartMigration")
		const school: SchoolDO = await this.schoolService.getSchoolById(schoolId);
		const migratedUsers: Page<UserDO> = await this.userService.findUsers({
			schoolId,
			outdatedSinceEquals: school.oauthMigrationFinished,
		});

		migratedUsers.data.forEach((user: UserDO) => {
			user.outdatedSince = undefined;
		});
		await this.userService.saveAll(migratedUsers.data);

		school.oauthMigrationMandatory = undefined;
	}
			console.timeEnd("restartMigration")
```